enhancements/oc/must-gather.md

```diff
@@ -110,9 +114,10 @@ If the `oc adm must-gather` tool's pod cannot be scheduled or run on the cluster
 
 ### Implementation Details/Notes/Constraints
 
-What are the caveats to the implementation? What are some important details that
-didn't come across above? Go in to as much detail as necessary here. This might
-be a good place to talk about core concepts and how they releate.
+During the implementation of `--since` and `--since-time`, `--until` and
+`--until-time` were also considered.
+Supporting `until` flags considerably increase the complexity of the task, so more
+research is required before pursuing it.
```

Comment:
Worth mentioning the limitation of the rotated logs and linking https://github.com/openshift/oc/pull/1653.

Reply:
Good idea, updated.